als-structure/shared/src/test/resources/RAML10/structure/test002/api-outline.json

```diff
@@ -421,142 +421,7 @@
             "character": 0
           }
         },
-        "children": [
```

Comment:
are Abstract children not shown on purpose?

Reply:
The abstract DataNode specifically. Maybe we could work with the data node as the operation/endpoint that it represent and show it as those are shown? Probably out of scope for this adoption though